pandas/core/apply.py

```diff
@@ -191,7 +191,7 @@ def apply_broadcast(self, target):
 
         for i, col in enumerate(target.columns):
             res = self.f(target[col])
-            ares = np. asarray(res).ndim
```

Comment:
Are we even hitting this block of code if we managed to get away with that!?

Reply:
I haven't looked at the apply tests closely, but I was somewhat surprised this is line works with the whitespace...

```
In [1]: import numpy as np

In [2]: np. asarray([1, 2, 3]).ndim
Out[2]: 1

In [3]: np. asarray([1, 2, 3])
Out[3]: array([1, 2, 3])

In [4]: np. asarray([1, 2, 3]).ndim == np.asarray([1, 2, 3]).ndim
Out[4]: True
```

Reply:
That's really odd!  I didn't know this could work.  And I thought Python was really picky about whitespace...

BTW, the easiest way to check is to deliberately raise an exception in that logic block and run tests with that bugged install.

Reply:
Looks like Python actually allows for you to put spaces after periods and will just evaluate as if the spaces weren't there, so this may or may not actually be getting hit:

```python
In [7]: np.   array([0, 2, 1]).   max()
Out[7]: 2
```

Reply:
Looks like the following tests in `pandas/tests/frame/test_apply.py` hit this path:

```
TestDataFrameApply.test_apply_broadcast_deprecated
TestDataFrameApply.test_apply_broadcast
TestDataFrameApply.test_apply_broadcast_error
TestInferOutputShape.test_result_type

```